install.py

```diff
@@ -352,10 +341,6 @@ def validate_path(path):
         cmake_flags += ["-Dcunumeric_cuRAND_INCLUDE_DIR=%s" % curand_dir]
     if legate_dir:
         cmake_flags += ["-Dlegate_core_ROOT=%s" % legate_dir]
```

Comment:
This is now always non-None.

Reply:
https://github.com/nv-legate/cunumeric/pull/656/commits/16facef7e98b65e1e0d32f522b0bb8a9acf2a40c